test/integration/testsuite_test.go

```diff
@@ -76,25 +92,12 @@ var _ = BeforeSuite(func() {
 
 	Expect(err).NotTo(HaveOccurred())
 
-	bundlePath = os.Getenv("BUNDLE_PATH") // this env var should contain location of bundle
 	if bundlePath != "" {
 		Expect(bundlePath).To(BeAnExistingFile())
 	}
 
-	ginkgoOpts = os.Getenv("GINKGO_OPTS")
-	if err != nil {
-
-		logrus.Infof("Error: Could not read GINKGO_OPTS.")
-		logrus.Infof("%v", err)
-	}
-	Expect(err).NotTo(HaveOccurred())
-
-	pullSecretPath = os.Getenv("PULL_SECRET_PATH") // this env var should contain location of pull-secret file
-	if err != nil {
-
-		logrus.Infof("Error: You need to set PULL_SECRET_PATH to find CRC useful.")
-		logrus.Infof("%v", err)
+	if pullSecretPath == "" {
```

Comment:
Sorry did not see this before, for golang this is commonly done:

```golang
if len(pullSecretPath) == 0 {....
```

As so in case the var at some point is set to nil it will pass on this condition, but for the equal empty string will throw panic error

Reply:
I see. Let me fix it.